src/core/module.ts

```diff
@@ -152,7 +161,7 @@ export const coreModule = (): Module<CoreModule> => ({
       baseQuery,
       reducerPath,
       context,
-      api,
+      api: api as any,
```

Comment:
I couldn't resolve the TS error here, and I'm not too sure why being that the types should match all the way through.